src/test/java/org/fundacionjala/sfdc/stepdefinition/AccountSteps.java

```diff
@@ -0,0 +1,124 @@
+        salesForceMainTab = new SalesForceMainTabClassic(webDriver, "a.accountBlock", "li#AllTab_Tab");
+        salesForceMainTab.displayOptions();
+        salesForceMainTab.goToAccounts();
+    }
+
+    /**
+     * When step.
+     */
+    @When("^I press new account button a new account form is displayed$")
+    public void iPressNewAccountButtonANewAccountFormIsDisplayed() {
+        System.out.println("This Step presses new account button");
+        webDriver = DriverManager.getInstance().getNavigator();
+        salesForceAccounts = new SalesForceAccounts(webDriver);
+        Assert.assertTrue(salesForceAccounts.isNewAccountButtonDisplayed());
+    }
+
+    /**
+     * And step.
+     */
+    @And("^I fill the account name field and press the save button$")
+    public void iFillTheAccountNameFieldAndPressTheSaveButton() {
+        System.out.println("This Step fills account name field");
+        webDriver = DriverManager.getInstance().getNavigator();
+        propertiesManager = PropertiesManager.getInstance().getConfig();
+        salesForceAccounts = new SalesForceAccounts(webDriver);
+        salesForceAccounts.createNewAccount(propertiesManager.getProperty("accountName"));
+        salesForceAccounts.clickSaveNewAccountButton(salesForceAccounts.getSaveNewAccountButton());
+    }
+    /**
+     * Then step.
+     */
+    @Then("^a new account is created$")
+    public void aNewAccountIsCreated() {
+        webDriver = DriverManager.getInstance().getNavigator();
+        propertiesManager = PropertiesManager.getInstance().getConfig();
+        salesForceAccounts = new SalesForceAccounts(webDriver);
+        System.out.println("This Step tests new account creation:" + salesForceAccounts.newAccountSavedName());
+        Assert.assertEquals(salesForceAccounts.newAccountSavedName(), propertiesManager.getProperty("accountName"));
+
+    }
+    /**
+     * When step.
+     */
+
+    @When("^I choose an account from recent accounts and I click on Edit account$")
+    public void iChooseAnAccountFromRecentAccountsAndIClickOnEditAccount() {
+        System.out.println("Choose last account");
+        webDriver = DriverManager.getInstance().getNavigator();
+        salesForceAccounts = new SalesForceAccounts(webDriver);
+//        String param = propertiesManager.getProperty("accountView");
+        //salesForceAccounts.chooseAccounts(param, salesForceAccounts.getGoButton());
+        salesForceAccounts.clickAccountNameLink();
+        salesForceAccounts.clickEditAccount();
+    }
+    /**
+     * When step.
+     */
+    @When("^I choose an account from recent accounts and I click on Delete account$")
+    public void iChooseAnAccountFromRecentAccountsAndIClickOnDeleteAccount() {
+        System.out.println("Choose last account");
+        webDriver = DriverManager.getInstance().getNavigator();
+        salesForceAccounts = new SalesForceAccounts(webDriver);
+      //  String param = propertiesManager.getProperty("accountView");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)